jobs/cloud_controller_ng/templates/bin/cloud_controller_ng.erb

```diff
@@ -7,16 +7,21 @@ wait_for_blobstore
 
 cd /var/vcap/packages/cloud_controller_ng/cloud_controller_ng
 
+<% def out(s); s.strip; end %>
```

Comment:
I haven't understood why this function is needed... below is an "if" with "echo"s and external script calls in the body. Why does this not work the "skip_validation" check?

Reply:
Indeed, the function is not needed. Thank you for the hint.